scripts/core/superdesk-api.d.ts

```diff
@@ -3674,10 +3679,20 @@ declare module 'superdesk-api' {
 
         userOnlineMinutes: number;
 
-        // e.g. {nl: 'leuven_dutch'}
-        spellcheckers?: {
-            [languageCode: string]: string;
-        };
+        spellchecking?: {
```

Comment:
we should document those api changes somewhere

Reply:
I think it might be still useful to keep some changelog for api, you would see it fail but you wouldn't know how to fix it from those errors probably